### DISCUSSION TITLE
[광고] visionOS 앱 개발 공부하고 싶은 사람?

### DISCUSSION THREAD

**Discussion by manju-minji** [DeveloperAcademy-POSTECH/.github#75] [Show and tell]
visionOS 앱 개발의 기초 지식을 쌓을 수 있는 강의를 만들었습니다.

Vision Pro 실 기기 없이 시뮬레이터만 이용하여 학습이 가능합니다!
많관부!!

[강의 바로 가기](https://www.udemy.com/course/basic_visionos/?referralCode=9EA3CC0133D8477FF125)

**Comment by yeniful** [comment]
강사님이 친절하고 강의가 맛있어요!
